libs/core/src/lib/facets/content/facet-content.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, ChangeDetectionStrategy, ViewEncapsulation, ElementRef, Renderer2, OnInit } from '@angular/core';
+import { FACET_CLASS_NAME } from '../constants';
+import { addClassNameToFacetElement } from '../utils';
+
+@Component({
+    selector: 'fd-facet-content',
+    template: ` <ng-content></ng-content> `,
```

Comment:
Just a question, not asking to change it :) Why did you choose to make this a component and not a directive? If it's a directive you will "expose" the native element to the user, for example `<div>` and will allow them to add props, classes, etc.

Reply:
If we have these as directives, we would have the extra effort of wrapping these into components again in platform(to support metaui) without any major enhancement. 😅  Also, in case of form and rating indicator facet where `fd-facet-content` is used, any extra classes can still be applied at the `fd-facet-content` tag level and no special inputs are present for this component.